trojsten/submit/views.py

```diff
@@ -423,12 +441,7 @@ def task_submit_post(request, task_id, submit_type):
                 if task.email_on_code_submit:
                     send_notification_email(sub, task_id, submit_type)
 
-                success_message = format_html(
-                    "Úspešne si submitol program, výsledok testovania nájdeš "
-                    '<a href="{}">tu</a>',
-                    reverse("view_submit", args=[sub.id]),
-                )
-                messages.add_message(request, messages.SUCCESS, success_message)
+                return redirect(reverse("view_submit", args=[sub.id]))
```

Comment:
Je toto nieco co chceme? Ja by som to dal do zvlast PR, aby sa to dalo reviewovat samostatne. Nie som 100% presvedceny ze toto je to co ludia chcu.

Reply:
Je to jeden cieľov toho issue.